packages/components/src/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -80,7 +82,12 @@ export const SegmentedControlItem = (
     { children, ...itemProps }: SegmentedControlItemProps,
     ref: ForwardedRef<HTMLButtonElement>,
 ) => (
-    <ToggleGroupPrimitive.Item ref={ref} {...itemProps} className={segmentedControlItemStyles} asChild={false}>
+    <ToggleGroupPrimitive.Item
+        ref={ref}
+        {...itemProps}
+        className={cn(segmentedControlItemStyles, FOCUS_OUTLINE)}
```

Comment:
please add in `segmentedControlItemStyles`

Reply:
fixed